Makefile

```diff
@@ -462,6 +462,10 @@ eve: $(BIOS_IMG) $(EFI_PART) $(CONFIG_IMG) $(PERSIST_IMG) $(INITRD_IMG) $(ROOTFS
 	cp pkg/eve/build.yml pkg/eve/runme.sh images/*.yml $|
 	$(PARSE_PKGS) pkg/eve/Dockerfile.in > $|/Dockerfile
 	$(LINUXKIT) pkg $(LINUXKIT_PKG_TARGET) --disable-content-trust --hash-path $(CURDIR) --hash $(ROOTFS_VERSION)-$(HV) $(if $(strip $(EVE_REL)),--release) $(EVE_REL) $(FORCE_BUILD) $|
+ifeq ($(EVE_REL),snapshot)
```

Comment:
I really don't think this logic belongs in a Makefile -- ultimately it is a hack for a very particular way of how some of our test infrastructure seems to be using our releases and as such -- lets just add it as a one-off in the publish action (where the rest of the hacks like making sure both build, etc. etc. belong)

Reply:
I uploaded a new version with the docker commands in publish.yml instead.